OMEdit/OMEditLIB/Simulation/SimulationDialog.cpp

```diff
@@ -2322,7 +2324,7 @@ DataReconciliationDialog::DataReconciliationDialog(LibraryTreeItem *pLibraryTree
   }
   mpDataReconciliationMeasurementInputFileTextBox->setText(mpLibraryTreeItem->mSimulationOptions.getDataReconciliationMeasurementInputFile());
   mpDataReconciliationCorrelationMatrixInputFileTextBox->setText(mpLibraryTreeItem->mSimulationOptions.getDataReconciliationCorrelationMatrixInputFile());
-  mpDataReconciliationEpsilonTextBox->setText(mpLibraryTreeItem->mSimulationOptions.getDataReconciliationEpsilon());
+  mpDataReconciliationEpsilonTextBox->setText("1.e-10");
```

Comment:
This is not good. You should set the default value where `mpDataReconciliationEpsilonTextBox` is initialized.